recipes/tensorflow-lite/all/test_v1_package/conanfile.py

```diff
@@ -0,0 +1,18 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "arch", "compiler", "build_type"
+    generators = "cmake", "cmake_find_package_multi"
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def test(self):
+        if not tools.cross_building(self):
+            model_path = os.path.join(self.source_folder, "..", "test_package", "model.tflite")
```

Comment:
```suggestion
            model_path = os.path.join(self.source_folder, os.pardir, "test_package", "model.tflite")
```

Please use `os.pardir`.